network-api/networkapi/utility/faker/streamfield_provider.py

```diff
@@ -113,7 +170,11 @@ def streamfield(self, fields=None):
             'image': generate_image_field,
             'spacer': generate_spacer_field,
             'image_text2': generate_image_text2_field,
-            'quote': generate_quote_field
+            'quote': generate_quote_field,
```

Comment:
Should I arrange this list alphabetically?

Reply:
the CMS presents them in the order that the code specifies - I don't think we ever really thought about what the best ordering is, so I don't think anyone would object. Pinging @kristinashu just to make sure

Reply:
Sorry which order are you talking about it? If it's about the quote component there's actually a bug that multiple quotes don't show up https://github.com/mozilla/foundation.mozilla.org/issues/2845

Reply:
Oh you mean this? For consistency in finding things it would be nice if they were in the same order as "opportunity" pages (minus the ones we chose not to included):
<img width="728" alt="image" src="https://user-images.githubusercontent.com/1068783/57249697-78297200-6ffa-11e9-90f7-e4f280095873.png">

Reply:
![Artboard 1](https://user-images.githubusercontent.com/2896608/57333311-5f3fc000-70d1-11e9-9e13-ff05d25a96e2.png)

Alright. Looks like we are good here!